release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -73,6 +73,11 @@ With this enhancement, users can set the `--import-mode` flag to `Legacy` or `Pr
 
 //Insert reference for feature procedure
 
+[id="ocp-4-13-oc-describe-enhancement"]
+==== Returning os/arch and digests of an image
+
+With {product-title} {product-version}, running `oc describe` on an image now returns os/arch and digests of each manifest.
```

Comment:
I'm not sure if it needs to be said, but I coulnd't tell: Do you run `oc describe` at the cli or the install shell?